src/Microsoft.DotNet.Build.Tasks.Packaging/src/Generations.cs

```diff
@@ -16,203 +16,6 @@ namespace Microsoft.DotNet.Build.Tasks.Packaging
 {
     public class Generations
     {
-        /// <summary>
-        /// A cache of path to generation definition file > Generations objects to avoid 
```

Comment:
Yeah, I kept it in a separate commit.  It was just some cleanup.  If you prefer I can merge it in a separate PR.